CMakeLists.txt

```
@@ -17,22 +18,45 @@ option(DOWNLOAD_ENSMALLEN "If ensmallen is not found, download it." ON)
 option(DOWNLOAD_STB_IMAGE "Download stb_image for image loading." ON)
 option(BUILD_GO_SHLIB "Build Go shared library." OFF)
 option(BUILD_DOCS "Build doxygen documentation (if doxygen is available)." ON)
+set(BOARD_NAME "" CACHE INTERNAL "Enter the board name to compile mlpack for.")
+
+if(CMAKE_CROSSCOMPILING)
+  include(board/flags-config.cmake)
+  if(BOARD_NAME STRGREATER  "")
+    flags(${BOARD_NAME})
+  endif()
+endif()
```

```suggestion
if (CMAKE_CROSSCOMPILING)
  include(board/flags-config.cmake)
  if (BOARD_NAME STRGREATER  "")
    flags(${BOARD_NAME})
  endif()
endif()
```

Might be a good idea to use the same style across the file, I think above you used another style as well.

---

I will unify it locally